src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Presentation/SampleChooserViewModel.cs

```diff
@@ -776,13 +805,16 @@ void Dispose(object snd, RoutedEventArgs e)
 				container.DataContext = null;
 			}
 
+			Console.WriteLine("UpdateContent GetRecentSamples");
 			var recents = await GetRecentSamples(ct);
 
+			Console.WriteLine("UpdateContent GetCategory");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6226524)